internal/status-tracker/src/validation_info.rs

```diff
@@ -11,17 +11,26 @@
 // specific language governing permissions and limitations under
 // each license.
 
+//! Describes a signature's validation information.
+
 use c2pa_crypto::SigningAlg;
 use chrono::{DateTime, Utc};
 use x509_parser::num_bigint::BigUint;
 
 #[derive(Debug, Default)]
 pub struct ValidationInfo {
-    pub alg: Option<SigningAlg>, // validation algorithm
+    /// Algorithm used to validate the signature
+    pub alg: Option<SigningAlg>,
```

Comment:
I don't _love_ this solution, but perhaps convert `SigningAlg` to `String`? @mauricefisher64 @gpeacock any other ideas? This introduces a circular dependency between c2pa-crypto and c2pa-status-tracker.

I suppose another alternative which I didn't previously consider is to move `ValidationInfo` into c2pa-crypto instead. Intra-crate circular dependencies are OK.